nio/event_builders/state_events.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2018, 2019 Damir Jelić <poljar@termina.org.uk>
+#
```

Comment:
I believe you should add your own copyright here as well.

Reply:
What is the format for multiple authors? Do I just add a `Copyright © 2019 miruka <adress>` line?

Reply:
Yeah, you just add a new line.